colourgame.html

```diff
@@ -7,29 +7,40 @@
   </body>
 <script>
 var guess,
-  answer,
-  clue1,
-  clueguess;
+  	answer,
+  	response,
+  	color,
+  	number;
+
+number = Math.floor(Math.random() * 10);
+
+if (number < 3) {
+  color = "Red";
+
+} else if (number > 3 && number < 7) {
+  color = "Blue";
+
+} else if (number > 7) {
+  color = "Green";
+
+};
```

Comment:
the above selection isn't quite balance (randomness-wise). 
red and blue are about a third more likely than green. 
they span four numbers (3 to 0 and 7 to 3) while green only has a range of 3 (10 to 7).